README.md

```diff
@@ -1,12 +1,30 @@
-# Webpack Boilerplate
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
->  A simple yet powerful webpack boilerplate, which you can later use as a starting point in your projects.
+# ToDo App
 
+>  A basic  web app that allows you to add todo task.
 
-## Features
 
-- Webpack setup 
-- HtmlWebpackPlugin
-- Loading CSS setup
-- Webpack-dev-server
+## Built With
 
+- HTML, CSS, Javascript
+- Linters
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+[Live Demo](https://hannziegel.github.io/captstoneModule1/)
+
+## Getting Started
+Clone the repository or use the code button on the right to download the zip file of the project, unzip it and open the index.html file in the browser
+
```

Comment:
- [ ] Kindly update the "Getting Started" section. Currently, this way your project is not running. This should be an example of how you may give instructions on setting up your project locally. :+1: